iOS/issue-tracker/issue-tracker/Network/LoginHelper.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import Combine
+
+class LoginHelper {
+    
+    private let callbackURLScheme = "issueTracker"
+    private let gitHubLoginUrlString = "https://github.com/login/oauth/authorize?client_id=04fb3475fc652d5304a3"
+    
+    func getCallbackURLscheme() -> String {
+        return callbackURLScheme
+    }
+    
+    func getGitHubLoginURL() -> URL? {
+        return URL(string: gitHubLoginUrlString)
+    }
```

Comment:
오 분리하니 깔끔하고 좋네요 👍
또 다른 선택지로는 연산프로퍼티도 있음을 알려드려요! 
저는 함수 인수가 없고 반환값이 있는 경우는 연산프로퍼티가 깔끔해서 자주 쓰는 편입니다. 
또 아시겠지만 추가적으로 말씀드리면 연산 프로퍼티는 var 로 적혀있지만 반환했을 때 read-only 프로퍼티라 값을 바꿀 수 없는 상수 값입니다.   

> https://docs.swift.org/swift-book/LanguageGuide/Properties.html

```
Read-Only Computed Properties
A computed property with a getter but no setter is known as a read-only computed property. A read-only computed property always returns a value, and can be accessed through dot syntax, but can’t be set to a different value.

NOTE

You must declare computed properties—including read-only computed properties—as variable properties with the var keyword, because their value isn’t fixed. The let keyword is only used for constant properties, to indicate that their values can’t be changed once they’re set as part of instance initialization.

You can simplify the declaration of a read-only computed property by removing the get keyword and its braces:

struct Cuboid {
    var width = 0.0, height = 0.0, depth = 0.0
    var volume: Double {
        return width * height * depth
    }
}
let fourByFiveByTwo = Cuboid(width: 4.0, height: 5.0, depth: 2.0)
print("the volume of fourByFiveByTwo is \(fourByFiveByTwo.volume)")
// Prints "the volume of fourByFiveByTwo is 40.0"
This example defines a new structure called Cuboid, which represents a 3D rectangular box with width, height, and depth properties. This structure also has a read-only computed property called volume, which calculates and returns the current volume of the cuboid. It doesn’t make sense for volume to be settable, because it would be ambiguous as to which values of width, height, and depth should be used for a particular volume value. Nonetheless, it’s useful for a Cuboid to provide a read-only computed property to enable external users to discover its current calculated volume.
```

Reply:
감사합니다! 참고하겠습니다!